src/services/query.ts

```diff
@@ -21,13 +21,18 @@ export interface ConstructorOptions {
   distribution: Distribution;
   query: string;
   comunica: IActorInitSparqlArgs;
+  timeoutMs: number;
 }
 
 const schemaConstructor = Joi.object({
   logger: Joi.object().required(),
   distribution: Joi.object().required(),
   query: Joi.string().required(),
   comunica: Joi.object().required(),
+  timeoutMs: Joi.number()
+    .integer()
+    .max(parseInt(process.env.MAX_QUERY_TIMEOUT as string) || 10000)
```

Comment:
Or `process.env.MAX_QUERY_TIMEOUT as number`?

Reply:
That doesn’t work:

```
TS2352: Conversion of type 'string | undefined' to type 'number' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.   Type 'string' is not comparable to type 'number'.
```

Reply:
👍🏻